### PR TITLE
OrgSettings endpoints

### PR DESCRIPTION
**Pull request by ElliotThiebaut**
Added OrgSetting endpoint to fetch public settings.
Added related types.
Updated packages

### CODE REVIEW

File: src/models/OrgSettings.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  message: string;
+  dateRange: DateRange;
+}
+
+export interface DateRange {
+  startDate: string;
+  endDate: string;
+}
+
+export enum ServiceStatus {
```
Comment (ElliotThiebaut) [inline src/models/OrgSettings.ts:17]: @charliefoxtwo I've put placeholders in the enum, what should each one be named to be consistent with what the API expects?

File: src/models/OrgSettings.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+export default interface OrgSettings {
+  status: ServiceStatus;
+  emergenciesEnabled: boolean;
+  messageOfTheDay: MessageOfTheDay;
```
Comment (charliefoxtwo) [inline src/models/OrgSettings.ts:4]: nullable

File: src/models/OrgSettings.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+export interface MessageOfTheDay {
+  message: string;
+  dateRange: DateRange;
```
Comment (charliefoxtwo) [inline src/models/OrgSettings.ts:9]: nullable

File: src/models/OrgSettings.ts (hunk 1 above)
Comment (charliefoxtwo) [reply src/models/OrgSettings.ts:17]: iirc, should be:
```
UNKNOWN,
HEALTHY,
SLIGHTLY_DEGRADED,
HEAVILY_DEGRADED,
OFFLINE,
```

**Review by charliefoxtwo — CHANGES_REQUESTED** [review]

File: src/models/OrgSettings.ts (hunk 3 above)
Comment (ElliotThiebaut) [reply src/models/OrgSettings.ts:9]: In the date range are startDate and endDate also nullable? Or just dateRange generaly?

Comment (charliefoxtwo) [reply src/models/OrgSettings.ts]: just `dateRange` - we decided that if a start date is provided an end date must also be provided and vice versa right?

Comment (ElliotThiebaut) [reply src/models/OrgSettings.ts]: Ok perfect thats the current implementation

**Review by charliefoxtwo — APPROVED** [review]

**Pull request merged**